R/plumb-block.R

```diff
@@ -207,6 +208,11 @@ plumbBlock <- function(lineNum, file){
       comments <- paste(comments, commentMat[1,2])
     }
 
+    routerModifierMat <- stri_match(line, regex="^#['\\*]\\s*@plumber(.*)$")
+    if (!is.na(routerModifierMat[1,2])) {
```

Comment:
I don't want to imply that there is something to be captured.

```suggestion
    routerModifierMat <- stri_match(line, regex="^#['\\*]\\s*@plumber")
    if (!is.na(routerModifierMat[1,1])) {
```